p4include/v1model.p4

```diff
@@ -55,6 +55,8 @@ limitations under the License.
 #define V1MODEL_VERSION 20180101
 #endif
 
+const bit<32> __v1model_version = V1MODEL_VERSION;
+
```

Comment:
On my machine, putting this in the beginning of the file causes it to misbehave, and output this in the dump files (incorrectly), which is why I put it at the end originally...

Reply:
I will put it in the middle. But it would be nice to understand why this happens.
The input from cpp seems correct.